packages/orchestration/src/orchestration-util.ts

```diff
@@ -0,0 +1,51 @@
+import type { DocumentGroundingServiceConfig } from './orchestration-types.js';
+import type {
+  AzureContentSafety,
+  GroundingModuleConfig,
+  InputFilteringConfig,
+  OutputFilteringConfig
+} from './client/api/schema/index.js';
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param filter - Filtering configuration for Azure filter. If skipped, the default Azure content filter configuration is used.
+ * @returns An object with the Azure filtering configuration.
+ */
+export function buildAzureContentFilter(
+  filter?: AzureContentSafety
+): InputFilteringConfig | OutputFilteringConfig {
+  if (filter && !Object.keys(filter).length) {
+    throw new Error('Filter property cannot be an empty object');
+  }
+  return {
+    filters: [
+      {
+        type: 'azure_content_safety',
+        ...(filter && { config: filter })
```

Comment:
[req] @KavithaSiva I saw this was your code change. Why are we expanding `filter` here as well instead of just writing

```ts
filters: [
  {
    type: 'azure_content_safety',
    config: filter
  }
]

Reply:
I'll create a followup ticket for this. This function needs to be fixed anyway since it's incorrect to return a filters array from here anyway. Will also include the point you made here